content/source/docs/enterprise/admin/agents-on-tfe.html.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: "enterprise"
+page_title: "Terraform Cloud Agents on TFE - Infrastructure Administration - Terraform Enterprise"
```

Comment:
Can you add a metadata page description to the top of this page? :-) It helps with SEO!

Reply:
Is that a `description` field? I hadn't seen that on other pages, I'm assuming all that was added in https://github.com/hashicorp/terraform-website/pull/1841

Reply:
yes! And you're right, it's new :-)  Maybe something like: 

description: |-
   Cloud agents let Terraform manage isolated, private, or on-premises infrastructure. Learn what's different when you use cloud agents in Terraform Enterprise vs. Terraform Cloud.

Reply:
Added! Thanks!